client/my-sites/site-settings/form-general.jsx

```diff
@@ -653,35 +653,16 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
-	disablePrivacySettings = ( e ) => {
-		e.target.blur();
-	};
-
 	privacySettingsWrapper() {
 		if ( this.props.isUnlaunchedSite ) {
-			if ( this.props.needsVerification ) {
-				return (
-					<EmailVerificationGate>
-						{ this.renderLaunchSite() }
-						{ this.privacySettings() }
```

Comment:
we've been working on this as part of coming soon, so my first instinct is to say we should feature toggle this behind the coming-soon-v2 feature toggle e.g. by doing something like:
```
	 { ! config.isEnabled( 'coming-soon-v2' ) && this.privacySettings() }
```

But maybe it's fine to just make this change for everyone, that would certainly make the implementation simpler and allow us to get that refactoring in.